lib/Bailador/App.pm

```diff
@@ -327,7 +349,13 @@ class Bailador::App does Bailador::Routing {
                     }
                 }
                 default {
+                    Log::Any.error(
+                      .gist,
+                      :extra-fields( Hash.new( ( $env.kv, :file-and-line($?FILE~':'~$?LINE), :pid($*PID), :client-ip('-') ) ) ),
+                      :pipeline('web')
+                    );
```

Comment:
Do we really need to calles of Log::Any.error(.gist, ...)?

Reply:
For now, it's logged in a file, or nowhere if the file is not selected. I kept this line to keep the error logged in the console. But maybe the error should be logged to STDERR by default and so we could keep both comportments in only one call.